numba/testing/main.py

```diff
@@ -579,7 +579,6 @@ def add_results(self, result):
         self.expectedFailures.extend(result.expectedFailures)
         self.unexpectedSuccesses.extend(result.unexpectedSuccesses)
 
-
```

Comment:
This is deleted line should be put back!?

Reply:
Good catch